src/Symfony/Component/Console/Helper/Helper.php

```diff
@@ -170,7 +170,7 @@ public static function removeDecoration(OutputFormatterInterface $formatter, ?st
         // remove <...> formatting
         $string = $formatter->format($string ?? '');
         // remove already formatted characters
-        $string = preg_replace("/\033\[[^m]*m/", '', $string);
+        $string = preg_replace("/\033\[[^m]*m/", '', $string ?? '');
```

Comment:
why call `preg_replace` on an empty string?
```suggestion
        $string = $string ? preg_replace("/\033\[[^m]*m/", '', $string) : '';
```

Reply:
Right, we can save the call here.